content/guides/code-splitting-import.md

```diff
@@ -144,6 +144,28 @@ module.exports = {
 };
 ```
 
+## `import` imports the entire module namespace
+
+Consider the following two examples:
+
+```js
+// Example 1: top-level import
+import * as Component from './component';
+// Example 2: Code-splitting with import()
+import('./component').then(Component => /* ... */);
+```
+
+`Component` in both of those cases resolves to the same thing, meaning in the case of using `import()` with ES2015 modules you have to explicity access default and named exports:
+
+```js
+async function main() {
+  // Destructuring example
+  const { default: Component } = await import('./component');
```

Comment:
This was just to show the same thing two different ways. One gets a reference to the component, and the other passes it inline. This can be broken apart if confusing.